src/cli/list/jobs/Jobs.definition.ts

```diff
@@ -29,7 +29,7 @@ export const ListJobsDefinition: ICommandDefinition = {
         },
     ],
     options: [{
-        name: "prefix", aliases: ["p"],
+        name: "prefix", aliases: [],
```

Comment:
For future reference, we should be careful when removing/modifying the how users interact with the puglin since it could be considered a breaking change. (No big deal for now)

Reply:
The alias "p" was added by mistake, I think. It conflicts with the alias "p" of the option password.